tools/check-changelog-entry-file/main.go

```diff
@@ -53,12 +55,34 @@ func validateChangelog(filePath, body string) {
 	entry := changelog.Entry{
 		Body: body,
 	}
-	if err := entry.Validate(); err != nil {
-		log.Fatalf("Error validating changelog file: %s, err: %v", filePath, err)
+	notes := changelog.NotesFromEntry(entry)
```

Comment:
knit: it might be worth it to have a comment pointing to the original code so we don't forget to check from time to time to see if they made it more extensible and can use it

Reply:
added a small comment giving this context